coverage/tomlconfig.py

```diff
@@ -0,0 +1,146 @@
+import io
+import os
+import re
+
+from coverage import env
+from coverage.backward import configparser, path_types, string_class, toml
+from coverage.misc import CoverageException, substitute_variables
+
+
+class TomlDecodeError(Exception):
+    """An exception class that exists even when toml isn't installed."""
+
+
+class TomlConfigParser:
+    def __init__(self, our_file):
```

Comment:
We don't have a case where our_file is True, so we could remove a little bit of code here.